src/RuleDictionnaryDropdownCollection.php

```diff
@@ -56,7 +56,7 @@ public function replayRulesOnExistingDB($offset = 0, $maxtime = 0, $items = [],
             return $this->replayRulesOnExistingDBForModel($offset, $maxtime);
         }
 
-        if (isCommandLine()) {
+        if (isCommandLine() && !defined('TU_USER')) {
```

Comment:
I would prefer to not have to add this.

In tests, you can catch output using `$this->output(...);` syntax.